contrib/colab/PyTorch_TPU_XRT_1_13.ipynb

```diff
@@ -0,0 +1,563 @@
+        "y = torch.rand(3, 5)                                                                                                    \n",
+        "model = XlaMulAdd()                                                                                                     \n",
+        "traced_model = torch.jit.trace(model, (x, y))                                                                             \n",
+        "xla_model = torch_xla._XLAC.XlaModule(traced_model)                                                             \n",
+        "output_xla = xla_model((torch_xla._XLAC.XLATensor(x), torch_xla._XLAC.XLATensor(y)))                                               \n",
+        "expected = model(x, y)\n",
+        "print(output_xla[0][0].to_tensor().data)\n",
+        "print(expected.data)\n"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "PM8Iljy-OXhx",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 4683
+        },
+        "outputId": "26ad236f-7cb0-4778-9042-1bbc1100d69a"
+      },
+      "cell_type": "code",
+      "source": [
+        "import shutil\n",
+        "import time\n",
+        "import torch\n",
+        "import torch.nn as nn\n",
+        "import torch.nn.functional as F\n",
+        "import torch.optim as optim\n",
+        "from torchvision import datasets, transforms\n",
+        "import torch_xla\n",
+        "import torch_xla_py.utils as xu\n",
+        "import torch_xla_py.xla_model as xm\n",
+        "import unittest\n",
+        "import sys\n",
+        "import os\n",
+        "import argparse\n",
+        "\n",
+        "parser = argparse.ArgumentParser(add_help=False)\n",
+        "parser.add_argument('--datadir', type=str, default='/tmp/mnist-data')\n",
+        "parser.add_argument('--logdir', type=str, default='/tmp/logs')\n",
+        "parser.add_argument('--num_cores', type=int, default=1)\n",
+        "parser.add_argument('--batch_size', type=int, default=512)\n",
+        "parser.add_argument('--num_epochs', type=int, default=10)\n",
+        "parser.add_argument('--num_workers', type=int, default=4)\n",
+        "parser.add_argument('--target_accuracy', type=float, default=98.0)\n",
+        "parser.add_argument('--fake_data', action='store_true')\n",
+        "parser.add_argument('--tidy', action='store_true')\n",
+        "parser.add_argument('--metrics_debug', action='store_true')\n",
+        "\n",
+        "FLAGS, leftovers = parser.parse_known_args()\n",
+        "sys.argv = [sys.argv[0]] + leftovers\n",
+        "# Setup import folders.\n",
+        "xla_folder = os.path.dirname(os.path.dirname(os.path.abspath(sys.argv[0])))\n",
+        "print('xla folder', xla_folder)\n",
+        "sys.path.append(os.path.join(os.path.dirname(xla_folder), 'test'))\n",
+        "sys.path.insert(0, xla_folder)\n",
```

Comment:
A lot of the code above can be removed.
That is only in place to have the tests select our local dev files, instead of the installed ones.
A bunch of those arguments above can be removed, and the code simplified.
Like, logdir, fake_data, tidy, metrics_debug.
Code must be simplified to the bone to give the impression it is as close as possible to vanilla PT.
And all the options above simply add stuff which is confusing for a user.